package.json

```diff
@@ -4,7 +4,7 @@
   "desktopName": "ferdium.desktop",
   "appId": "org.ferdium.ferdium-app",
   "version": "6.5.0-nightly.8",
-  "description": "Messaging app for WhatsApp, Slack, Telegram, HipChat, Hangouts and many many more.",
```

Comment:
could you also please delete the hipchat recipe in this same PR if the service has been discontinued?

Reply:
sorry, I don't have this repo cloned on my computer and I don't think I can delete a folder from the recipes using the GitHub Web UI.

Reply:
looks like I cannot add your fork as a new remote (for me to make the changes and push). I will update in a new branch and make the changes - but, it will mean that I need to close your PR.

Reply:
no problem - thanks for sorting. 👍